cli/internal/turbopath/anchored_unix_path.go

```diff
@@ -25,9 +25,9 @@ func (p AnchoredUnixPath) ToUnixPath() AnchoredUnixPath {
 }
 
 // RelativeTo calculates the relative path between two `AnchoredUnixPath`s.
-func (p AnchoredUnixPath) RelativeTo(basePath AnchoredUnixPath) (AnchoredUnixPath, error) {
+func (p AnchoredUnixPath) RelativeTo(basePath AnchoredUnixPath) (AnchoredSystemPath, error) {
```

Comment:
It looks like the callsites immediately convert to `AnchoredUnixPath`. Maybe this function should do that, rather than change types?

Reply:
The correct answer here is actually to avoid triggering `FromSlash` or `ToSlash` as those calls are cross-platform unsafe. It's _usually_ not a problem because most people on POSIX systems don't create file names like `this\is\going\to\go\badly.txt` but it's an unfortunate edge case.

I want to _actually_ fix this, but it's not something I can justify doing in our current world.

In the mean time I'm going to drop this function and address the call sites with string manipulation.